Source/Fuse.Controls.ScrollView/Tests/UX/WhileVisibleInScrollView.ux

```diff
@@ -9,6 +9,12 @@
 		<WhileVisibleInScrollView ux:Name="V3" Distance="1" RelativeTo="ScrollViewSize">
 			<Nothing/>
 		</WhileVisibleInScrollView>
+		<WhileVisibleInScrollView ux:Name="V4" How="Full">
```

Comment:
I'd prefer if this were split off into a separate test. Different features are good to separate when possible.